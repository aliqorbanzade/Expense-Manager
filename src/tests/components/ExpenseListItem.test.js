import React from 'react'
import expenses from '../fixtures/expenses'
import {shallow} from 'enzyme'
import ExpenseListItem from '../../components/ExpenseListItem'

test('shoul render exp list item',()=>{
    const wrapper = shallow(<ExpenseListItem {...expenses[1]}/>)

    expect(wrapper).toMatchSnapshot()
})