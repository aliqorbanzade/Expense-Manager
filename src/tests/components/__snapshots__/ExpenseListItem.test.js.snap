// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shoul render exp list item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    $1,095.00
     - 
    December 28O , 1969
  </p>
</div>
`;
